docs/tutorials/vc-jwt/vc-jwt.collection.json

```diff
@@ -156,7 +156,7 @@
 				"header": [
 					{
 						"key": "Accept",
-						"value": "application/json",
```

Comment:
I would remove this part

Reply:
Could you say more? In this case the `Accept` header is on a `Verify` request, so just represents a preference that the verification response be in json format, right?